okteto-stack.yml

```diff
@@ -1,22 +1,24 @@
 name: voting-app
+
+endpoints:
```

Comment:
if this is a getting started example, wouldn't it be easier to go with `public: true`?  I think the `endpoints` key should be reserved for the scenarios where it provides clear value (e.g. specify a non-standard route)

Reply:
The problem is that you need to explain what `public` is and how it works.
`endpoints` (level 7 proxy) is already a concept for developers, it is less magic to explain

Reply:
Endpoints and a proxy are two different concepts. I think it's going to be hard for someone to read 'endpoint' and realize that it is the equivalent of doing a proxy to one of their services. I think that making a service `public` is a lot easier to grasp for someone who's getting started with stacks  (that is the whole reason why that feature exists in the first place no?).

Reply:
I have kept it with `public ` for now.
But I keep getting the same feedback from people that having several ways to do the same thing is very confusing

Reply:
100% agree. if we want to only support endpoints that's ok too, we just need to make sure is well documented.